elements/lisk-transactions/src/register_delegate.ts

```diff
@@ -49,21 +56,37 @@ export const registerDelegate = (
 	inputs: RegisterDelegateInputs,
 ): Partial<TransactionJSON> => {
 	validateInputs(inputs);
-	const { username, passphrase, networkIdentifier } = inputs;
+	const {
+		username,
+		passphrase,
+		networkIdentifier,
+		passphrases,
+		keys,
+		senderPublicKey,
+	} = inputs;
 
 	const transaction = {
 		...createBaseTransaction(inputs),
 		type: 10,
+		// For txs from multisig senderPublicKey must be set before attempting signing
+		senderPublicKey: senderPublicKey ? senderPublicKey : undefined,
```

Comment:
type `senderPublicKey` is `string | undefined` so this equation is not necessary?

Reply:
Removed.